src/spectra/blackbody.cpp

```diff
@@ -108,7 +120,7 @@ class BlackBodySpectrum final : public Texture<Float, Spectrum> {
                        lambda5 = sqr(lambda2) * lambda;
 
             // Wien's approximation to Planck's law
-            Wavelength pdf = 1e-9f * c0 * exp(-c1 / (lambda * m_temperature))
+            Wavelength pdf = 1e-9f * m_scale * c0 * exp(-c1 / (lambda * m_temperature))
```

Comment:
Do we really need to scale the PDF here? I think those values should be "normalized" no? Is this how it is implemented in Mitsuba 0.6?